site/_scss/blocks/_deprecation-calendar.scss

```diff
@@ -0,0 +1,262 @@
+    font-size: 1rem;
+    padding: 1rem 0;
+
+    @include media-query('md-max') {
+      background-color: var(--color-footer-bg);
+      border-radius: 0.75rem;
+      flex-wrap: wrap;
+      margin-bottom: 1.5rem;
+    }
+
+    a {
+      text-decoration: none;
+    }
+
+  }
+
+  &__row > div:first-child {
+    padding-left: 1rem;
+    width: 16%;
+    
+    @include media-query('md-max') {
+      font-size: px-to-rem(14px);
+      margin-top: 1.5rem;
+      order: 1;
+      padding: 0 1.5rem;
+      width: 50%;
+    }
+  }
+
+  &__row > div:nth-child(2) {
+    width: 60%;
+
+    @include media-query('md-max') {
+      padding: 0 1.5rem;
+      width: 100%;
+    }
+
+    @include media-query('md') {
+      margin: 0 px-to-rem(24px);
+    }
+  }
+
+  &__row > div:nth-child(3) {
+    text-align: center;
+    width: 12%;
+
+    @include media-query('md-max') {
+      font-size: px-to-rem(14px);
+      margin-top: 1.5rem;
+      order: 1;
+      padding: 0 1.5rem;
+      text-align: left;
+      width: 50%;
+    }
+    
+    @include media-query('md') {
+      margin-right: px-to-rem(24px);
+    }
+  }
+
+  &__row > div:nth-child(4) {
+    padding-right: 1rem;
+    text-align: center;
+    width: 12%;
+
+    @include media-query('md-max') {
+      border-top: 1px solid get-color('grey-300');
+      margin: 1.5rem;
+      margin-bottom: 0;
+      order: 2;
+      padding-top: 1.5rem;
+      width: 100%;
+    }
+  }
+
+  &__row:nth-child(even) {
+    background-color: var(--color-footer-bg);
+    border-radius: 4px;
+
+    @include media-query('md-max') {
+      border-radius: 0.75rem;
+    }
+  }
+
+  &__header {
+    font-weight: 500;
+
+    @include media-query('md-max') {
+      display: none;
+    }
+
+  }
+
+  &__mobile-label {
+    display: none;
+
+    @include media-query('md-max') {
+      display: block;
+    }
+  }
+
+  &__article {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    margin: 0 auto;
+    width: px-to-rem(30px);
+
+    @include media-query('md-max') {
+      justify-content: start;
+      width: auto;
+    }
+
+    :hover {
+      background-color: transparent;
+    }
+
+    svg {
+      margin-right: 8px;
+    }
+
+  }
+
+  .hero-card {
+    margin: 0;
+    max-width: 100%;
+  }
+
+  .hero-card__inner {
+    @include media-query('md-max') {
+      text-align: center;
+    }
+
+    @include media-query('md') {
+      display: grid;
+      grid-template-columns: 2fr 3fr;
+    }
+  }
+
+  &__filters {
+
+    align-items: center;
+    display: flex;
+    justify-content: start;
+    padding: 1rem 0;
+
+    @include media-query('md-max') {
+      display: none;
+    }
+
+    > span {
+      font-size: 1rem;
+      font-weight: 500;
+      margin-right: 1rem;
+    }
+    
+    .enhanced-select__wrapper {
+      border: 0;
+      color: get-color('blue-600');
+      font-weight: 500;
+    }
+
+    .enhanced-select label {
+      margin-left: 1rem;
+    }
+
+    .enhanced-select__options {
```

Comment:
Styling for `enhanced-select` should be consistent across all pages, so ideally we would not need to overwrite anything here.